src/lib/charts/ChartLegend/ChartLegend.tsx

```diff
@@ -1,66 +1,33 @@
-import { ChartType } from 'chart.js';
-import React, { FC, HTMLAttributes } from 'react';
-import {
-  ChartJSData,
-  ChartJSDataFunction,
-  ChartLegendComponent,
-  ChartRefObject,
-  getPieChartPercentage
-} from '../chart.helpers';
+import classNames from 'classnames';
+import React, { FC } from 'react';
+import { ChartJSData, ChartLegendComponentProps } from '../chart.helpers';
+import { ChartLegendItem } from './ChartLegendItem';
 
-import './chart-legend.css';
+export const ChartLegend: FC<ChartLegendComponentProps> = ({ data, interactive, onItemClick }) => {
+  // TODO: Add support for pie charts
 
-export interface ChartLegendProps extends HTMLAttributes<HTMLDivElement> {
-  type?: ChartType;
-  data: ChartJSData | ChartJSDataFunction;
-  chartRef: ChartRefObject;
-  component?: ChartLegendComponent;
-}
+  const { datasets } = data as ChartJSData;
 
-const PieChartLegendContent: FC<{ data: ChartJSData }> = ({ data }) => {
-  if (!data.datasets || !data.datasets[0].data || !data.labels) return null;
-
-  const dataset = data.datasets[0];
-
-  // Note: I'm reversing the data here so that we can have the farthest left item align with the furthest left item on
-  // the chart, which seems to make the most sense. - Jake 03/16/2021
-  const reversedData = [...(dataset.data as number[])].reverse();
-  const reversedLabels = [...(data.labels as string[])].reverse();
-  const reversedBackgroundColors = [...(dataset.backgroundColor as string[])].reverse();
+  if (!datasets) return null;
 
   return (
-    <ul className="lc-chart-legend-list">
-      {reversedData.map((value, index) => {
```

Comment:
FYI, I moved your pie chart legend into the `PieChart` component file.

I'm going to clean it all up in a subsequent PR, but for now, this should be good enough.